pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computetargethttpsproxy/globaltargethttpsproxy/create.yaml

```diff
@@ -20,6 +20,8 @@ spec:
   description: "test description"
   urlMapRef:
     name: computeurlmap-${uniqueId}
+  certificateMapRef:
```

Comment:
According to the field description: https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/8d78e4962662663dbbb3de8262c6e12ed14e398f/config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_computetargethttpsproxies.compute.cnrm.cloud.google.com.yaml#L64

`sslCertificates and certificateManagerCertificates fields can not be defined together.`

And according to the latest TF doc: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_target_https_proxy#certificate_manager_certificates

`Certificate manager certificates only apply when the load balancing scheme is set to INTERNAL_MANAGED. For EXTERNAL and EXTERNAL_MANAGED, use certificate_map instead.`

Feels like `certificateManagerCertificates` and `certificateMap` are mutually exclusive fields, and `certificateManagerCertificates`/ `certificateMap` and `sslCertificates` are mutually exclusive fields as well. I.e. only one out of the three fields can be configured. 

Could you verify that certificate map is indeed configured correctly in the underlying GCP resource when ssl certificates are configured?

Once the behavior is confirmed, could you update the field descriptions of those fields to reflect the correct usage of these fields?

Reply:
The default value for `load balancing schemes` is EXTERNAL, so I'm able to created a target https proxy with `certificateMap`: https://screenshot.googleplex.com/7Lm35rNZ6hkmXMZ.png

However, `certificateManagerCertificates`/`certificateMap` and `sslCertificates` are mutually exclusive fields and I noticed that `sslCertificates` is discarded when both are set. I updated the test and field description.